test/fft_utest.cpp

```diff
@@ -140,6 +144,68 @@ class FftTest : public ::testing::Test {
             ASSERT_EQ(ifft_1, v);
         }
     }
+
+    /**
+     * convert a number into a vector of digits padded with zeros
+     *
+     * @param num
+     *
+     * @return
+     */
+    quadiron::vec::Vector<T>*
+    _convert_string2vec(const gf::Field<T>& gf, int n, char num[])
+    {
+        int i;
+        quadiron::vec::Vector<T>* vec = new quadiron::vec::Vector<T>(gf, n);
+        int len = strlen(num);
+        for (i = 0; i < len; i++) {
+            vec->set(i, num[len - i - 1] - '0');
+        }
+        for (; i < n; i++) {
+            vec->set(i, 0);
+        }
+        return vec;
+    }
+
+    /** compute sum of 2 large numbers in string representation
+     *
+     * @param str1 first string
+     * @param str2 second string
+     *
+     * @return the sum string
+     */
+    std::string big_num_add(std::string str1, std::string str2)
+    {
+        if (str1.length() > str2.length())
+            std::swap(str1, str2);
+
+        std::string str = "";
+
+        int n1 = str1.length(), n2 = str2.length();
```

Comment:
One declaration per line + `length` probably returns `size_t` so:

```
size_t n1 = str1.length();
size_t n2 = str2.length();
```

But anyway, getting the length is O(1) (not O(n) like `strlen`), you don't can use them in the `for` loop directly (the compiler will optimize).